### ISSUE TITLE
Java sets not working as intended?  

### ISSUE THREAD

**Issue by oeystein** [stoicflame/enunciate#292]
Been setting up enunciate on an existing project i have over the past days, and randomly came across that one of my attributes were missing from the generated docs. After some research i discovered that it occurred to all my Set Attributes. The expected behavior would be that Sets should just be interpreted as any Collection/Array/List etc. But they just dont show appear at all. 

For instance the following yields no result in the generated API Docs. 
```
@XmlElement(name = "device")
@XmlElementWrapper(name = "devices")
private Set<UUID> devices = null;
```

**Comment by stoicflame** [comment]
Weird. Thanks for the report.

**Comment by oeystein** [comment]
Was a double issue at my end, and not really Set related, but rather strictness of enunicate. So sorry for any confusion i may have caused.

One of the issues i met was that one class created an json attribute by getters and setters, which Getter was fine, and returned a Set like it were supposed to, but the Setter took in an Collection instead of a Set, which was apparently works fine for both Jax-B and FasterXML. 

```
    @NotNull
    @XmlElement(name = "add")
    @XmlElementWrapper(name = "adds")
    public Set<UUID> getAdditions() {
        return additions;
    }

    @NotNull
    public UuidChangeSet setAdditions(@NotNull Collection<UUID> additions) {
        this.additions.addAll(additions);
        return this;
    }
```

An other one was that someone had used the XMLElement annotation tag in the @JsonPropertyOrder rather then the attribute name, which also worked fine for both of Jax-B and FasterXML.  

```
@JsonPropertyOrder(...,  "device")
class Device{
@XmlElement(name = "device")
@XmlElementWrapper(name = "devices")
private Set<UUID> devices = null;
...
}
```

**Comment by stoicflame** [comment]
I fixed the issue with the unpaired setter at 06c3440. Thanks for the report.

I couldn't see the problem you reported with the use of `@JsonPropertyOrder`, so I'll close this out. If you'd like to provide me more details on how to see the issue, go ahead and open up a new issue.

**Comment by stoicflame** [comment]
[Enunciate 2.3 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.3.0).
